### PR TITLE
Hotfix for aggregation

### PR DESCRIPTION
**Pull request by lntuition**
We found #317 change `joins` order in aggregation. So removing reversed operation may fix this bug(and fortunately, it leads to better performance) Add more testcase to check aggregation. 

Special thank to detailed bug reporting @pjongy 
Fixed : #327 

- [x] Fix bug in join with aggregation
- [x] Add more testcase to aggregation 
- [x] Add Changelog

### CODE REVIEW

**Comment by lntuition** [comment]
@pjongy Could you check that this PR fix your problem?

**Comment by pjongy** [comment]
Thank you for your quick response and hotfix.
It seems bug is fixed and looks good to me! :)

File: tests/test_aggregation.py [hunk 1]
```diff
@@ -50,6 +50,23 @@ class TestAggregation(test.TestCase):
...
+
+        await event.participants.add(team_second)
+        await event.participants.add(team_first)
+
+        tournaments = await Tournament.annotate(
+            events_participants_count=Count("events__participants")
+        ).filter(id=tournament.id)
+        self.assertEqual(len(tournaments), 1)
+        self.assertEqual(tournaments[0].id, tournament.id)
+
```
Comment (pjongy) [inline tests/test_aggregation.py]: I've copied exsitence test code for skeleton so... we need to check about our requirements
I think check assert for `events_participants_count` realy `2` is reasonable
``` python
self.assertEqual(tournaments[0].events_participants_count, 2)
```

**Comment by pjongy** [comment]
Oh, I've reviewed something minor can you check it?

File: tests/test_aggregation.py (hunk 1 above)
Comment (lntuition) [reply tests/test_aggregation.py]: Good point. I fixed it

**Pull request merged**